demos/WindowsPowerShellModules/README.md

```diff
@@ -0,0 +1,54 @@
+# Using Windows PowerShell modules with PowerShell Core
+
+## Windows PowerShell vs PowerShell Core
+
+Existing Windows PowerShell users are familiar with the large number of modules available, however, they are not necessarily compatible with PowerShell Core.
+More information regarding compatibility is in a [blog post](https://blogs.msdn.microsoft.com/powershell/2017/07/14/powershell-6-0-roadmap-coreclr-backwards-compatibility-and-more/).
+
+Windows PowerShell 5.1 is based on .Net Framework 4.6.1, while PowerShell Core is based on .Net Core 2.0.
+Although both adhere to .Net Standard 2.0 and can be compatible, some modules may be using APIs not supported on CoreCLR or using APIs from Windows PowerShell that have been deprecated and removed from PowerShell Core (for example, PSSnapins).
+
+## Importing a Windows PowerShell module
+
+Since compatibility cannot be ensured, PowerShell Core, by default, does not look in the Windows PowerShell module path to find those modules.
+However, for advanced or adventurous users, they can explicitly enable PowerShell Core to include the Windows PowerShell module path and attempt to import those modules.
+
+First, install a module from the PowerShellGallery to enable this:
```

Comment:
Reword  " ... a module ..." to " ... WindowsPSModulePath module from the PowerShell Gallery<insert link to the module> ..."

Reply:
Will fix